linetools/analysis/linelimits.py

```diff
@@ -25,8 +28,9 @@ class LineLimits(object):
     vlim : Quantity array
       velocity limits for the line in km/s
     """
+
     @classmethod
-    def from_specline(cls, aline, zlim):
+    def from_specline(cls, aline, z, zlim):
```

Comment:
No.  aline.z is a pointer to LineLimits._z
That is the point here, i.e. we want one redshift value set and used.

It does require these to be defined, but they are defaulted to 0 and (0,0)